cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -187,4 +187,17 @@ describe('Top toolbar tests.', function() {
 		cy.get('#copy-paste-container table')
 			.should('exist');
 	});
+
+	it('Insert image.', function() {
+		cy.get('#tb_editbar_item_insertgraphic')
+			.click({force: true});
```

Comment:
There are two things here.
1. We don't need to click on this item, because we can't test the file picker directly anyway. What we should test here, whether this toolbar item is visible.
2. I guess you used {forced: true} because this item is hidden by default. There is a scroll item (w2ui-scroll-right) with which you can scoll on the top toolbar and make this item visible.